src/shogun/classifier/LDA.cpp

```diff
@@ -49,9 +49,7 @@ void CLDA::init()
 	SG_ADD(
 	    (machine_int_t*)&m_method, "m_method",
 	    "Method used for LDA calculation", MS_NOT_AVAILABLE);
-	SG_ADD(
-	    (machine_int_t*)&m_gamma, "m_gamma", "Regularization parameter",
-	    MS_NOT_AVAILABLE);
+	SG_ADD(&m_gamma, "m_gamma", "Regularization parameter", MS_NOT_AVAILABLE);
```

Comment:
I think this might break the integration tests (did you check?)
If so, pls generate a new test data file

Reply:
no test broken :)

Reply:
i think gamma should be model selectable